src/pages/crm-products-page/CrmProductsPage.tsx

```diff
@@ -108,8 +106,8 @@ const CrmProductsPage = () => {
   }, [searchDebounce])
 
   useEffect(() => {
-    setPaginationModel((model) => ({
-      ...model,
+    setPaginationModel(() => ({
+      pageSize: PAGE_SIZE,
```

Comment:
maybe you mean this one?
```
setPaginationModel({...paginationModel, page: searchDebounce ? searchProductsForCrmPageNumber : productsForCrmPageNumber})

Reply:
I am confused, which line do I need to fix

Reply:
every part where you have `useCallback` and setting state as a function
- remove `useCallback`
- set state as a object or other types, don't use function as a setter
- test this updates that all works okay

Reply:
I removed all useCallbacks and check everything
it works